src/content/configuration/optimization.md

```diff
@@ -69,11 +69,21 @@ By default webpack v4+ provides new common chunks strategies out of the box for
 
 `object` `string` `boolean`
 
-Setting `optimization.runtimeChunk` to `true` adds an additional chunk to each entry point containing only the runtime.
-It is possible to use preset mode of the plugin by providing a string value:
+Setting `optimization.runtimeChunk` to `true` or `"multiple"` adds an additional chunk to each entrypoint containing only the runtime. This setting is an alias for:
 
-- `single`: creates a runtime file to be shared for all generated chunks.
-- `multiple`: creates multiple runtime files for common chunks.
+```js
+runtimeChunk: {
```

Comment:
example snippets must be in full path and with filename otherwise we get issues raised saying example snippets arent nonice-friendly, e.g. this one should be:

`__webpack.config.js__`

```javascript
module.exports = {
  //...
  optimization: {
    runtimeChunk: {
      name: entrypoint => `runtime~${entrypoint.name}`
    }
  }
}
```